lib/node_modules/@stdlib/math/base/napi/unary/include/stdlib/math/base/napi/unary/d_f.h

```diff
@@ -0,0 +1,81 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_MATH_BASE_NAPI_UNARY_D_F_H
+#define STDLIB_MATH_BASE_NAPI_UNARY_D_F_H
+
+#include <node_api.h>
+#include <assert.h>
+
+/**
+* Macro for registering a Node-API module exporting an interface invoking a unary function accepting a double-precision floating-point number and returning a single-precision floating-point number.
+*
+* @param fcn   unary function
+*
+* @example
+* static float scale( const double x ) {
+*     return x * 10.0f;
+* }
+*
+* // ...
+*
+* // Register a Node-API module:
+* STDLIB_MATH_BASE_NAPI_MODULE_D_F( scale );
```

Comment:
```suggestion
* static float fcn( const double x ) {
*     return (float)x;
* }
*
* // ...
*
* // Register a Node-API module:
* STDLIB_MATH_BASE_NAPI_MODULE_D_F( fcn );
```

It is slightly awkward to have a function which accepts a double, but then is scaled by a float which is promoted to double for the operation before being downcast to float for the return value.